profiles-daemon/README.md

```diff
@@ -0,0 +1,261 @@
+2022-10-12 19:34:16.210  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data Redis repositories in DEFAULT mode.
+2022-10-12 19:34:16.265  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 14 ms. Found 0 Redis repository interfaces.
+2022-10-12 19:34:17.422  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
+2022-10-12 19:34:17.451  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
+2022-10-12 19:34:17.452  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.65]
+2022-10-12 19:34:17.710  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
+2022-10-12 19:34:17.710  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2945 ms
+2022-10-12 19:34:19.721  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
+2022-10-12 19:34:19.745  INFO 1 --- [           main] c.c.c.e.p.ProfilesApplication            : Started ProfilesApplication in 7.628 seconds (JVM running for 9.35)
+```
+
+The redis connection details have been defined in the [SessionExample.java](../local-clients/src/main/java/com/c12e/cortex/examples/local/SessionExample.java) file
+for local connections and can be passed via [spark-conf.json](../cache-profile/src/main/resources/conf/spark-conf.json)`./main-app/build/tmp/test-data/sink-ds` for jobs that need to be run in cluster.
+More configurations can be found [here](https://github.com/RedisLabs/spark-redis/blob/master/doc/configuration.md).
+In this case
+
+### Run as a Skill
+
+### Prerequisites
+* Ensure that the Cortex resources exist, specifically the Cortex Project and Profiles (built) and cached([Cache Profile](../cache-profile/README.md)). **The underlying data source of the Profile does not need to exist.**
+* Update the Env vars to:
+  ```
+  export REDIS_HOST=cortex-redis-node-0.cortex-redis-headless.cortex.svc.cluster.local
+  export REDIS_PORT=6379
+  export REDIS_USER=default
+  export REDIS_PASSWORD=XXX
+  export SCALECOUNT=3
+  ```
+
+To Build and run the skill:
+1. Run the following make commands:
+```
+make deploy-daemon
+```
+
+To invoke the skill
+1. Use the below command to invoke the skill
+    ```
+    cortex skills invoke --params-file ./profiles-daemon/src/main/resources/payload.json pdaemon request --project <projectName>
+   
+   
+    {
+       "success": true,
+       "activationId": "f8520e95-5416-490c-9d52-a6cb108234cb"
+    }
+    ```
+   1. output for the above can be looked at be
+       ```
+       cortex agents get-activation f8520e95-5416-490c-9d52-a6cb108234cb --project <projectName>
```

Comment:
Suggested wording: `To view the output of the above skill activation, run the following command. Replace the activationId with the one returned from the previous step.`

(And separating input from output might be helpful)